pysat/instruments/sw_f107.py

```diff
@@ -521,14 +681,28 @@ def parse_daily_solar_data(data_lines):
         split_line = line.split()
 
         # Format the date
-        dates.append(pysat.datetime.strptime("".join(split_line[0:3]),
-                                                     "%Y%m%d"))
+        dfmt = "%Y %m %d" if year > 1996 else "%d %b %y"
+        dates.append(pysat.datetime.strptime(" ".join(split_line[0:3]), dfmt))
 
         # Format the data values
+        j = 0
         for i,kk in enumerate(val_keys):
-            val = split_line[i + 3]
+            if year == 1994 and kk == 'new_reg':
+                # New regions only in files after 1994
+                val = -999
+            elif((year == 1994 and kk in xray_keys) or
```

Comment:
missing space?
elif ((

Reply:
I don't think so:

> Avoid extraneous whitespace in the following situations: Immediately inside parentheses, brackets or braces.

Reply:
Oh, you're talking about a space before.  Ew.  I hope that isn't PEP8!